src/assets/sass/protocol/includes/forms/_lib.scss

```diff
@@ -0,0 +1,32 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// Import all the things!
+@import '../lib';
+
+/* forms
+- use padding as much as possible, margins don't collapse on form elements
+*/
+
+$form-red: #D70022;
```

Comment:
Should this be a token? red-60 or red-70 might be close enough. But then again, is it even appropriate to use brand colors for this kind of thing? And what do we do for a Mozilla theme (whenever I can finish that stuff). Maybe we need to have a small set of universal "UI colors" separate from the rest of the brand palettes (not for this PR of course).

Reply:
Yeah the notification colours are also not in the brand palette either so UI seems to be different. I'm just working with what I was given 🤷‍♀️

Reply:
No wonder I thought this was such a good idea, it's yours from a year ago: https://github.com/mozilla/protocol-tokens/issues/53